apps/wallet-mobile/src/features/Settings/useCases/changeAppSettings/ApplicationSettingsScreen.tsx

```diff
@@ -40,7 +40,7 @@ export const ApplicationSettingsScreen = () => {
   const {network} = useSelectedNetwork()
 
   const {data: screenShareEnabled} = useScreenShareSettingEnabled()
-  const displayScreenShareSetting = Platform.OS === 'android' && !isProduction
```

Comment:
@jorbuedo I dunno if this option is displayed on screen if prod.

Reply:
Right now it doesn't, that's the issue.

Reply:
🤦🏻 the var is `displayScreenShareSetting`